### ISSUE TITLE
Attack range perks don't work when manually casting auto-cast abilities

### ISSUE THREAD

**Issue by Draze22** [arcadia-redux/12v12#690]
Frost Arrows, Searing Arrows, Posion Attack, etc.

**Comment by lovro-m** [comment]
Let me guess, cast range also doesn't increase attack range but increases auto-cast :D

**Comment by ZLOY5** [comment]
Yes, isn't it how it works in dota?

**Comment by ZLOY5** [comment]
Wiki:
If the ability is cast manually, using the designated [Hotkey](https://dota2.fandom.com/wiki/Hotkey), the ability uses the cast range. When using [Autocast](https://dota2.fandom.com/wiki/Autocast), it uses the hero's attack range. The cast range of these abilities can be further increased by other attack range bonuses based on the hero's range type. However, it is not affected by cast range bonuses

**Comment by Draze22** [comment]
Yep, only attack range bonusses affect the manual cast range of auto-cast attack modifiers.
